CHANGELOG.md

```diff
@@ -1,5 +1,116 @@
 # Changelog
 
+## v6.0.0-rc - 2017-11-16
+
+### Breaking changes
+
+- Fix `-Verbose` to not override `$ErrorActionPreference`. (#5113)
+- Fix `Get-Item -LiteralPath a*b` to return error if `a*b` doesn't actually exist. (#5197)
+- Remove `AllScope` from most default aliases to reduce overhead on creating new scopes. (#5268)
+- Change `$OutputEncoding` to be `UTF8` without `BOM` rather than `ASCII`. (#5369)
+
+### Engine updates and fixes
+
+- Add char range overload to the `..` operator, so `'a'..'z'` returns characters from 'a' to 'z'. (#5026) (Thanks @IISResetMe!)
+- Remove `CommandFactory` because it serves no real purpose. (#5266)
+- Change to not insert line breaks to output (except for tables). (#5193)
```

Comment:
Reword as: Change to not insert line breaks at console window width to output (except for tables).

Reply:
Will do.